source/scenes/home/BuyList/BuyList.container.tsx

```diff
@@ -19,23 +19,36 @@ import BuyList from './BuyList';
 
 import { RootState } from 'state/store';
 import IProvidersState from 'state/providers/types';
+import IVaultState from 'state/vault/types';
 
 const BuyListContainer: FC = () => {
   
   const linkTo = useLinkTo();
   const { supportedAssets }: IProvidersState = useSelector((state: RootState) => state.providers);
+  const { activeWallet }: IVaultState = useSelector((state: RootState) => state.vault);
+  const assetsFiltered = Object.keys(supportedAssets?.data)
```

Comment:
The `?` here will return `undefined` if `supportedAssets` is falsey which will throw when passed to `Object.keys`